tests/bgp/test_traffic_shift.py

```diff
@@ -0,0 +1,275 @@
+import pytest
+import logging
+import json
+import time
+import ipaddr as ipaddress
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.utilities import wait_tcp_connection
+from jinja2 import Template
+import re
+
+pytestmark = [
+    pytest.mark.topology('t1')
+]
+
+logger = logging.getLogger(__name__)
+
+TS_NORMAL = "System Mode: Normal"
+TS_MAINTENANCE = "System Mode: Maintenance"
+TS_ERROR = "System Mode: Not consistent"
```

Comment:
Actually it is not an error. That means that TSC doesn't know the state exactly. Something was changed manually.

Reply:
Thanks. Updated.